### DISCUSSION TITLE
Was this ever tested on EL8+ and EFI/PXE without iPXE?

### DISCUSSION THREAD

**Discussion by killergoalie** [DSpeichert/netbootd#16] [Q&A]
Was this ever tested on EL8+ and EFI/PXE without iPXE?
Or was it primarily used for Ubuntu/Debian distros, since all of the examples use iPXE and Ubuntu?

I'm running into weird issues where I have to add mount paths for files that should get matched to the prefix, like grub.cfg. and other EFI/UEFI based pxe requirements.

**Comment by DSpeichert** [comment]
Not really, I have not tested with non-Ubuntu OSes.

Can you elaborate on what you're trying to serve and how it doesn't work?
